Registry.php

```diff
@@ -164,4 +165,11 @@ public function getEntityManagerForClass($class)
 
         return $this->getManagerForClass($class);
     }
+
+    public function reset() : void
+    {
+        foreach ($this->getManagerNames() as $managerName) {
```

Comment:
@alcaeus My array looks like this in `1.12.3`:

```
  ["default"]=>
  string(35) "doctrine.orm.default_entity_manager"
```

and ends so with:

> Doctrine %s Manager named "doctrine.orm.default_entity_manager" does not exist.

Exception.

any hint how this can happen?

Reply:
the `bin/console debug:container --parameter doctrine.entity_managers` does also return it this way

Reply:
See #1103.